Formula/figtree.rb

```diff
@@ -0,0 +1,21 @@
+class Figtree < Formula
+  desc "Graphical viewer of phylogenetic trees"
+  homepage "http://tree.bio.ed.ac.uk/software/figtree/"
+  url "https://github.com/rambaut/figtree/releases/download/v1.4.3/FigTree_v1.4.3.tgz"
+  sha256 "f497d4dd3a6d220f6b62495b6f47a12ade50d87dbd8d6089f168e94d202f937b"
+
+  # tag "bioinformatics"
```

Comment:
You can remove this line. It's not needed for `brewsci/bio`. Every formula is bioinformatics!